README.md

```diff
@@ -152,6 +152,10 @@ For the weekly report, future costs are estimated based on the active compute no
 
 You can also run `ruby get_latest_azure_prices.rb`, which will use an existing Azure project (which provides the required credentials for the API) to run this update to the prices on command. By setting up a cronjob to run this separately from the main files (for example, at the start of each day), wait times for generating Azure weekly reports can be dramatically reduced.
 
+## Enabling/disabling projects
+
+By default, projects are created in the 'active' state. To disable (or re-enable) a project, choose the appropriate option from the application entrypoint script. If a project is disabled, it will not have any daily or weekly reports sent. It will still, however, be listed when the `list` entrypoint is run.
```

Comment:
Worth mentioning here that reports also won't be generated if the project's start date is in the future, or end date in the past.